www/src/components/guidelines/color/overview.js

```diff
@@ -0,0 +1,171 @@
+import React from "react"
+import { capitalize } from "lodash-es"
+import range from "range"
+
+import { SectionHeading, SectionSubheading } from "../typography"
+
+import Swatch from "./swatch"
+
+import { getA11yLabel, getTextColor } from "../../../utils/guidelines/color"
+import { copyColumnGutter, CopyColumn } from "../containers"
+import palette from "../../../utils/guidelines/extend-palette-info"
+import { Box, Flex } from "../system"
+import theme from "../../../utils/guidelines/theme"
+
+// Color swatches
+const swatchWidth = 40
+const swatchStyle = {
+  flexGrow: 0,
+  flexShrink: 0,
+  height: swatchWidth,
+  marginBottom: theme.space[1],
+  marginTop: theme.space[1],
+  marginRight: theme.space[2],
+  position: `relative`,
+  textAlign: `center`,
+  width: swatchWidth,
+}
+const colorNumber = {
+  ...swatchStyle,
+  color: theme.colors.grey[50],
+  fontSize: theme.fontSizes[1],
+  transform: `rotate(-45deg)`,
+}
+
+const colores = node => {
+  let colors = []
+
+  Object.keys(palette[node].colors)
+    .sort((a, b) => b - a)
+    .forEach((color, index) => {
+      const c = palette[node].colors[color]
+
+      colors.push(
+        <Swatch
+          key={`${index}-${c.hex}`}
+          color={c}
+          contrast={c.contrast}
+          a11yLabel={getA11yLabel(c.a11y, true)}
+          textColor={getTextColor(c.contrast)}
+          name={c.name}
+          isBase={c.base}
+          swatchStyle={swatchStyle}
+        />
+      )
+    })
+
+  return colors
+}
+
+const Palette = ({ color, handler }) => {
+  const node = Object.keys(palette)
+    .filter(group => group === color)
+    .map(node => node)
+
+  return (
+    <Box
+      key={node}
+      display={{ xxs: `block`, lg: `flex` }}
+      mb={{ xxs: 4, md: 0 }}
+      css={{ alignItems: `center` }}
+    >
+      <Box
+        textAlign={{ lg: `right` }}
+        width={{ lg: `8rem` }}
+        mr={{ lg: copyColumnGutter }}
+      >
+        <Box
+          as="button"
+          bg="none"
+          p={0}
+          onClick={e => {
+            handler(e, node)
+          }}
+          css={{
+            border: 0,
+            cursor: `pointer`,
+            WebkitAppearance: `none`,
+            "&&:hover span, &&:focus span": {
+              borderColor: theme.colors.link.hoverBorder,
+              color: theme.colors.gatsby,
+            },
+          }}
+        >
+          <SectionSubheading
+            as="span"
+            fontFamily="header"
+            fontWeight={0}
+            mt={0}
+            mb={0}
+            fontSize={4}
+            color="link.color"
+            css={{
+              borderBottom: `1px solid ${theme.colors.link.border}`,
+            }}
+          >
+            {capitalize(node)}
```

Comment:
This would be a good spot for a title attribute, like `title="Open color modal"` or something. That way it would be exposed as text description in assistive tech and it would act as a tooltip for mouse users.

Reply:
✅ 82116ef